ci/jenkins/Jenkinsfile_website_release

```diff
@@ -0,0 +1,67 @@
+// -*- mode: groovy -*-
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// Jenkins pipeline
+// See documents at https://jenkins.io/doc/book/pipeline/jenkinsfile/
+
+// timeout in minutes
+max_time = 180
+
+node('utility') {
+  // Loading the utilities requires a node context unfortunately
+  checkout scm
+  utils = load('ci/Jenkinsfile_utils.groovy')
+  custom_steps = load('ci/jenkins/Jenkins_steps.groovy')
+}
+
+utils.assign_node_labels(utility: 'utility', linux_cpu: 'mxnetlinux-cpu')
```

Comment:
Since this is a release pipeline, it must be run as restricted job on restricted slaves. Regular slaves are not permitted to have any publish access.

@leezu @aaronmarkham

Reply:
@marcoabreu Just to clarify, this pipeline is more of a utility pipeline that is run manually to build artifacts corresponding to a specific version and package them in s3 zip file. It does not publish anything to the MXNet website. It automates the job that is otherwise required to be done manually by the release manager.
It is an improvement over the pipeline that was used to build v1.7 artifacts: http://jenkins.mxnet-ci-dev.amazon-ml.com/job/docs/job/1.7-test/ which used regular slaves as well. This PR aims to check-in the pipeline code and add a stage to package artifacts in s3 zip file.

Reply:
@marcoabreu Just like @mseth10 mentioned, this pipeline will only be used to build the new website artifacts and upload them to a private S3 bucket for internal use. Sorry that the file name `Jenkinsfile_website_release` might be confusing. I've changed it to `Jenkinsfile_website_upload`.

Reply:
None the less, you are giving the pipeline write permissions. Our unrestricted slaves are set up to specifically not be able to persist anything. 

Otherwise, somebody could go ahead, submit a PR which hijacks a pipeline. They could then go ahead and download illegal content from somewhere and host it in our s3 bucket. Do you want to be responsible for that?

Back then, the slaves were specifically set up with an instance profile which prohibited all these actions. You might want to engage in a security review to ensure everything is still safe.

Reply:
> None the less, you are giving the pipeline write permissions. Our unrestricted slaves are set up to specifically not be able to persist anything.

Agreed, we can move the pipeline to jenkins-ci prod account, where we have access to restricted nodes.

Reply:
@marcoabrew @mseth10 The pipeline has beed switched to restricted node and tested on jenkins-ci prod account.